tests/integration/README.md

```diff
@@ -0,0 +1,219 @@
+
+### Bulk-process multiple test cases
+
+All commands (`check`, `vet`, `run`, and `update`) can process multiple test cases, one after another. You can specify more than one directory as positional arguments of these commands.
+
+```bash
+# Run two test cases one after another.
+$ python3 ./tests/integration/run.py run ./test_case_a/directory ./test_case_b/directory
+```
+
+You can also use the `...` path wildcard to allow for discovering test case directories recursively under a root directory.
+
+```bash
+# Run all test cases discovered recursively under a directory.
+$ python3 ./tests/integration/run.py run ./all/cases/...
+```
+
+### Select a subset of test cases to run
+
+In certain cases, we can utilize the feature of tags to select a subset of test cases to run with the `run` command.
+
+Each test case can be attached with one or more tags in the yaml configuration. For example, you may find some of our test cases having the `docker` tag as follows.
+
+```yaml
+description: ...
+tags:
+- docker
+steps:
+- ...
+```
+
+We typically have the test cases for the container image being a subset of the test cases for the Macaron Python package. We can mark the test cases shared for both purposes with the `docker` tag. When we do integration testing for the container image, we can add the argument `--include-tag docker` to filter only test cases tagged with `docker`.
+
+```bash
+# Test the container image with test cases having the `docker` tag.
+$ python3 ./tests/integration/run.py run --include-tag docker ./all/cases/...
+```
+
+The `--include-tag` flag can be specified multiple times. A selected test case must contain all tags specified with the `--include-tag` flag.
+
+```bash
+# Test the container image with test cases having the `docker` tag.
+$ python3 ./tests/integration/run.py run --include-tag tag-a --include-tag tag-b ./all/cases/...
+```
+
+There is also the `--exclude-tag` flag. A selected test case must also not contain any tag specified with the `--exclude-tag` flag.
+
+```bash
+# Only run test cases not tagged with `npm`.
+$ python3 ./tests/integration/run.py run --exclude-tag npm ./all/cases/...
+```
+
+You can simply think of each `--include-tag`/`--exclude-tag` argument as adding an additional constraint that a selected test case must satisfy".
+
+### Debug utility script
+
+In case you want to debug the utility script itself, there is the verbose mode for all commands which can be enabled with the `-v/--verbose` flag.
+
+
+## Test Config Reference
+
+### Test case Schema
+
+* `description` (`string`, required): The description of the test case.
+* `tags` (`array[string]`, optional, default is `[]`): The tags of the test case. When the `--tag <tag>` argument is passed with the `run` command , only run test cases having the tag `<tag>`. (The `--tag` argument is typically used in combination with test case discovery using the `...` wildcard).
```

Comment:
I think `--tag` flag is not currently available for any command. Perhaps this hasn't been updated to the latest `--include-tag` and `--exclude-tag` flags.

Reply:
This is correct. I previously had `--include-tag` as `--tag`.

Reply:
Fixed in a9fb833b679adc0d7ff797622561375ea451c692.